### ISSUE TITLE
Error in eslint when editing router file

### ISSUE THREAD

**Issue by kimroen** [ember-cli/eslint-plugin-ember#509]
When editing a `router.js` file while running eslint with the `route-path-style` rule enabled, I always get an error thrown from eslint:

```
TypeError: Cannot read property 'value' of undefined
Occurred while linting /path-to-project/app/router.js:40
    at CallExpression (/path-to-project/node_modules/eslint-plugin-ember/lib/rules/route-path-style.js:42:41)
```

It's pointing to this line:
https://github.com/ember-cli/eslint-plugin-ember/blob/5e01f5b273ac3dac5f7d2224f0735f666c61f83e/lib/rules/route-path-style.js#L42

I'm assuming this is because I haven't finished writing the route declaration, but the code is valid javascript.

An example of the state of the router file when this error occurs:

```js
Router.map(function() {
  this.route()
});
```

More specifically I have typed `this.route`, and when I hit `(` the error is triggered (the editor automatically adds the `)` at this point).

**Comment by bmish** [comment]
Thanks for reporting, I'll fix this soon!
